ztp/resource-generator/Containerfile

```diff
@@ -1,4 +1,4 @@
-ARG ZTP_BUILD_IMAGE=registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.22-openshift-4.18
+ARG ZTP_BUILD_IMAGE=registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.23-openshift-4.19
```

Comment:
I don't think we can do this yet - The build image has to be rhel-8 based as long as we produce binaries (site-generator and pgt-generator) that need to run inside a rhel8 container (ArgoCD)